homeassistant/components/niu/translations/en.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Rename en.json -> strings.json

Reply:
Use key references in strings.json.
https://developers.home-assistant.io/docs/internationalization/core